### DISCUSSION TITLE
Experiment metrics charts and dashboard

### DISCUSSION THREAD

**Discussion by pauli31** [mlflow/mlflow#7315] [Q&A] [answered]
Hi,
is there a way how to get a dashboard in the web UI similar to the one in wandb (see picture).
I want to have an overview of different metrics and I would like to see them all at once
![image](https://user-images.githubusercontent.com/11619371/201311218-8f5c900f-af93-4895-bbf8-1aa7c12ae7ab.png)

Thank you!

**Comment by JHibbard** [comment, marked as answer]
Hi pauli31,

There's currently no dashboarding feature that's analogous to wandb's as pictured. But you could programmatically query the metrics of your choice from the tracking server and plot that data with your visualization tool of choice. This would also be a welcome feature if you're interested in OSS contributions.

Hope that helps!
Jim
